jinseok/model/userDao.js

```diff
@@ -19,8 +19,70 @@ const createUser = async (
       `,
       [email, profileImage, password, name, age, phoneNumber]
     );
-  } catch (err) {
-    console.log(err);
+  } catch (e) {
+    console.log(e);
+    const error = new Error('INVALID_DATA_INPUT');
+    error.statusCode = 500;
+    throw error;
+  }
+};
+
+const isExistedUser = async (email) => {
+  try {
+    const [result] = await dataSource.query(
+      `
+      SELECT EXISTS (
+        SELECT
+        id
+        FROM users 
+        WHERE email = ?
+    ) idExists
+    `,
+      [email]
+    );
+    return parseInt(result.idExists);
+  } catch (e) {
+    console.log(e);
+    const error = new Error('INVALID_DATA_INPUT');
+    error.statusCode = 500;
+    throw error;
+  }
+};
+
+const getUser = async (email) => {
```

Comment:
```suggestion
const getUserByEmail = async (email) => {
```
변수, 함수, 클래스, 데이터베이스 컬럼까지 포함하여 프로그래밍에서는 가급적 너무 길어지지 않는선에서 가장 가독성 있게 작성될 수 있으면 좋습니다.

아주 작성을 잘해주신 지금처럼 dao 함수는 CRUD(create, get, update, delete)와 목적리소스의 조합으로 이름을 짓되, 만약 email과 같은 특정 argument를 받아야한다면 `getUserByEmail`과 같은 식으로 지어주면 조금 더 좋습니다 :)